setup.py

```diff
@@ -7,17 +7,26 @@
     'eth': [
         "cryptography>=2.0.3,<3.0.0",
         "cytoolz>=0.9.0,<1.0.0",
-        "eth-bloom>=1.0.0,<2.0.0",
+        "eth-bloom==1.0.1",
```

Comment:
Any specific reason why the upper bound was removed?

Reply:
When using `eth-bloom>=1.0.0,<2.0.0`, the `mypy` check will fail as how #1710 is right now:
https://circleci.com/gh/ethereum/py-evm/124106?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
py35-lint runtests: commands[2] | mypy --follow-imports=silent --warn-unused-ignores --ignore-missing-imports --no-strict-optional --check-untyped-defs --disallow-incomplete-defs --disallow-untyped-defs --disallow-any-generics -p eth
eth/rlp/receipts.py:49: error: Cannot instantiate abstract class 'BloomFilter' with abstract attribute '__hash__'
eth/vm/base.py:415: error: Cannot instantiate abstract class 'BloomFilter' with abstract attribute '__hash__'
eth/vm/forks/frontier/blocks.py:67: error: Cannot instantiate abstract class 'BloomFilter' with abstract attribute '__hash__'
```

I think the reason why `master` branch *looks* fine now is due to Circle CI cache.

Reply:
Ah, just noticed. It's because of https://github.com/ethereum/py-evm/pull/1710